inference-engine/src/transformations/src/transformations/common_optimizations/add_fake_quantize_fusion.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/add_fake_quantize_fusion.hpp"
+#include "transformations/utils/utils.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include "itt.hpp"
+
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::AddFakeQuantizeFusion, "AddFakeQuantizeFusion", 0);
+
+ngraph::pass::AddFakeQuantizeFusion::AddFakeQuantizeFusion() {
+    MATCHER_SCOPE(AddFakeQuantizeFusion);
+    auto add_pattern = ngraph::pattern::wrap_type<opset5::Add>({ngraph::pattern::any_input(), ngraph::pattern::wrap_type<opset5::Constant>()},
```

Comment:
@jane-intel  can we now use `get_constant_from_input` inside transformation to avoid this dependencies on Constants?